### PR TITLE
BZ2105152: Memory Manager support status updated from TP to GA in 4.9

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s):
4.10

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2105152

Link to docs preview:
http://file.emea.redhat.com/amolnar/BZ2105152-4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-technology-preview

Additional information:

### CODE REVIEW

**Comment by fbaudin** [comment]
LGTM

**Comment by mrniranjan** [comment]
LGTM

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -2355,8 +2355,8 @@ In the table below, features are marked with the following statuses:
 
 |Memory Manager feature
```
Comment (mjpytlak) [inline release_notes/ocp-4-10-release-notes.adoc:2356]: Noticed that you removed `feature` in the respective 4.9 PR. https://github.com/openshift/openshift-docs/pull/47624.

For consistency, suggest removing `feature` here.

**Pull request merged**